common/djangoapps/util/date_utils.py

```diff
@@ -52,7 +53,7 @@ def get_time_display(dtime, format_string=None, coerce_tz=None):
         try:
             to_tz = timezone(coerce_tz)
         except UnknownTimeZoneError:
-            to_tz = UTC
+            to_tz = utc
         dtime = to_tz.normalize(dtime.astimezone(to_tz))
```

Comment:
I know you didn't write it, but this line looks problematic.  I need to delve back into the pytz docs to figure out what this should look like, but I think there's a problem with using both astimezone as normalize with the same tzinfo object (`to_tz`).  

Also, normalizing a datetime that isn't in UTC will raise an exception during ambiguous times (an hour before or after daylight savings stops or starts), meaning twice a year, this could potentially crash our app for users.